components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -208,6 +209,55 @@ protected String getUserNameWithoutChange() {
         return userNameWithoutChange;
     }
 
+    @Override
+    public void addSwaggerToLocalEntry(API api, String jsonText) {
+        if (log.isDebugEnabled()) {
+            log.debug("Adding a new Local Entry");
```

Comment:
Lets make this debug log more meaningful by adding the api too.

Reply:
Updated